Code/Framework/AzFramework/AzFramework/Script/ScriptComponent.cpp

```diff
@@ -963,6 +978,63 @@ namespace AzFramework
 
         return *this;
     }
+
+    // carbonated begin (akostin/mp226-2): Add NetBindable to ScriptComponent
+#if defined(CARBONATED)
+
+    const char* ScriptComponent::NetRPCFieldName = "NetRPCs";
+
+    //=========================================================================
+    // ScriptComponent::GetNetworkBinding
+    //=========================================================================
+    GridMate::ReplicaChunkPtr ScriptComponent::GetNetworkBinding()
+    {
+        if (m_netBindingTable == nullptr)
+        {
+            m_netBindingTable = aznew ScriptNetBindingTable();
+        }
+
+        return m_netBindingTable->GetNetworkBinding();
+    }
+
+    //=========================================================================
+    // ScriptComponent::SetNetworkBinding
+    //=========================================================================
+    void ScriptComponent::SetNetworkBinding(GridMate::ReplicaChunkPtr chunk)
+    {
+        if (m_netBindingTable == nullptr)
+        {
+            m_netBindingTable = aznew ScriptNetBindingTable();
+        }
+
+        m_netBindingTable->SetNetworkBinding(chunk);
+    }
+
+    //=========================================================================
+    // ScriptComponent::UnbindFromNetwork
+    //=========================================================================
+    void ScriptComponent::UnbindFromNetwork()
+    {
+        if (m_netBindingTable)
+        {
+            m_netBindingTable->UnbindFromNetwork();
+        }
+    }
+
+    //=========================================================================
+    // CreateNetworkBindingTable
+    // [6/27/2016]
```

Comment:
Is this line with data necessary?

Reply:
It is was copied from the original source, lets keep it.